### DISCUSSION TITLE
Add beta programmes.

### DISCUSSION THREAD

**Discussion by beedell-roke** [FossifyOrg/General-Discussion#123] [Ideas]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/General-Discussion/issues) or [closed](https://github.com/FossifyOrg/General-Discussion/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/General-Discussion/discussions) or [closed](https://github.com/FossifyOrg/General-Discussion/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs), in the [README](https://github.com/FossifyOrg/General-Discussion/blob/master/README.md) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report may get dismissed otherwise.**
- [X] This issue contains only one feature request.
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Which apps should have this feature?

All Fossify Apps

### Feature description

I'm aware that your FOSSify applications at the Play Store are in Early Access. However, this does not prevent you issuing a beta channel too.

Additionally, builds marked as betas (so that clients know potentially not to install them if the user has not enabled that) can be uploaded to the default F-Droid repository.

### Why do you want this feature?

This would provide the ability for users unable to compile for whatever reason to test new builds quicker.

### Additional information

Your CI will need to be able to work at the kind of speed necessary for that.

**Comment by beedell-roke** [comment]
> https://github.com/FossifyOrg/General-Discussion/issues/119#event-11861973388

@Aga-C, why the conversion?

**Reply by Aga-C** [reply]
Because we don't have beta releases currently, so it's rather an idea than a request.

**Comment by naveensingh** [comment]
> I'm aware that your FOSSify applications at the Play Store are in Early Access. However, this does not prevent you issuing a beta channel too.

"Early Access" aka Open testing _is_ the beta channel on the Google Play Store. It shows up as Early Access until there is at least one release on the Production track. Once there is a production release, you'll start seeing the message "You are a beta tester for this app. Future updates will include beta versions." (assuming you join the beta program).

Also, there won't be separate beta releases for other distribution channels because I don't see the need for it yet. Maybe in the future.

**Reply by beedell-roke** [reply]
> https://github.com/orgs/FossifyOrg/discussions/123#discussioncomment-8738275

@naveensingh, I was a beta tester for https://play.google.com/store/apps/details?id=com.microsoft.copilot&hl=en&gl=US whilst it was in pre-release. It merely appeared as "(Early Access) (Beta)". I admit multiple denominators looked goofy, but both channels existed simultaneously.

**Reply by naveensingh** [reply]
I have never seen such a channel on Google Play in the last 5 years or so. There are five release tracks (that I'm aware of): 

1. Production
2. Open (beta)
3. Closed (alpha, only for registered testers)
4. Internal (only for registered testers)
5. Pre-registration

They might just have been experimenting with the semantics or less likely but maybe there are more tracks that they expose through Google Play Console API.
